go/stellar/stellarsvc/frontend.go

```diff
@@ -197,3 +197,10 @@ func (s *Server) ChangeWalletAccountNameLocal(ctx context.Context, arg stellar1.
 
 	return stellar.ChangeAccountName(m, arg.AccountID, arg.NewName)
 }
+
+func (s *Server) DeleteWalletAccountLocal(ctx context.Context, arg stellar1.DeleteWalletAccountLocalArg) (err error) {
+	m := libkb.NewMetaContext(s.logTag(ctx), s.G())
+	defer s.G().CTraceTimed(ctx, "DeleteWalletAccountLocal", func() error { return err })()
```

Comment:
add
```
if err = s.assertLoggedIn(ctx); err != nil {
		return res, err
	}
```

Reply:
ah, thanks, i think i missed that on all my new rpcs